src/main/drivers/stm32/bus_i2c_hal_init.c

```diff
@@ -166,7 +166,7 @@ const i2cHardware_t i2cHardware[I2CDEV_COUNT] = {
         .device = I2CDEV_2,
         .reg = I2C2,
         .sclPins = { I2CPINDEF(PA9,  GPIO_AF4_I2C2), },
-        .sdaPins = { I2CPINDEF(PA10, GPIO_AF4_I2C2), },
+        .sdaPins = { I2CPINDEF(PA8, GPIO_AF4_I2C2), },
```

Comment:
SPRACINGH7FR uses PB10 for I2C2

Reply:
Just add it to the array. This will not be right, as I am not in front of a datasheet, but all the available pins should be added along with their AF.

e.g. `.sdaPins = { I2CPINDEF(PA8, GPIO_AF4_I2C2), I2CPINDEF(PB108, GPIO_AF4_I2C2), },`

Reply:
@haslinghuis This section I edited is G4 only; the MCU you're referring to uses different pin mappings.

Reply:
Added PF0.

Reply:
- edit: Added PF6, not PF0.